my-app/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
@RolandAmini You do not need all these files.
Do you setup in the `tokeners` directory